master/internal/rm/kubernetesrm/request_queue.go

```diff
@@ -16,19 +16,14 @@ const (
 	deletionGracePeriod  = 15
 )
 
-// callback type used by requestQueue.
```

Comment:
question: I see you're getting rid of the callback function here. Any plans to refactor out the pod/node/event/preemption informer callbacks from pods.go in future iterations?

Reply:
if it is necessary/better. the callback got very confusing in this case, that is why i removed it.